.changeset/weak-dolls-move.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cube-dev/ui-kit": patch
+---
+
+fix(menu): menu trigger behavior when inside modal
```

Comment:
Please, don't write changelog as you write your commits, prefer a human-oriented style instead, e.g. `Fixed bug when menu doesn't open within a modal`